checker/tests/src/tests/FenumTest.java

```diff
@@ -16,7 +17,7 @@ public FenumTest(File testFile) {
     }
 
     @Parameters
-    public static Collection<Object[]> data() {
-        return testFiles("fenum", "all-systems");
+    public static Collection<Object[]> getTestFiles() {
```

Comment:
That's a good suggestion.  I have done the following:

DefaultCheckerTest nows asks you to specify one of two methods:
```
@Parameters
public static void List<Object[]> getTestFiles() 
```
Which does the old behavior and allows people to find arbitrary test files (which is used in the InferenceTest suite to find all-systems in the checker-framework directory).
OR users can specify
```
@Parameters
public static void String[] getTestDirs() 
```
Which will return a list of directories that should be in the test directory.  The TestSuite will now enforce that a single method with one of the exact signatures above and an `@Parameters` has been added to your test file and otherwise it will explode with an informative exception/message.